webpack.config.js

```diff
@@ -0,0 +1,50 @@
+const webpack = require('webpack');
+const path = require('path');
+
+const config = {
+  entry: [
+    __dirname + '/client/src/index.jsx',
+  ],
+  output: {
+    path: path.resolve(__dirname, '/client/dist'),
+    filename: 'bundle.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        use: 'babel-loader',
+        exclude: /node_modules/
+      },
+      {
+        test: /\.css$/,
+        use: [
+          'style-loader',
```

Comment:
I notice that style-loader, css-loader and url-loader are all here in the module object, but not installed as dependencies.  Is that typical?

Reply:
@Gledinburgh Nice catch! CSS loader and URL loader were not whitelisted dependencies, so I was unsure about leaving that there for now and getting approval to use those packages( I believe we will have to if we are using any kind of custom CSS with webpack). But you are right, it's not typical to have loaders in the webpack.config file that you don't even have installed as dependencies.